src/popup.js

```diff
@@ -452,7 +453,7 @@ window.addEventListener('message', async (e) => {
                     note,
                     subject: callLogSubject,
                   }
-                  let callPage = logPage.getLogPageRender({ id: data.call.sessionId, manifest, logType: 'Call', triggerType: 'createLog', platformName, direction: data.call.direction, contactInfo: callMatchedContact ?? [], logInfo, loggedContactId: null });
+                  let callPage = await logPage.getLogPageRender({ id: data.call.sessionId, manifest, logType: 'Call', triggerType: 'createLog', platformName, direction: data.call.direction, contactInfo: callMatchedContact ?? [], logInfo, loggedContactId: null });
```

Comment:
@SushilMallRC Thinking it again. Do you reckon caching the produced page itself would be better?